Source/CesiumRuntime/Private/Tests/GlobeAwareDefaultPawn.spec.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020-2023 CesiumGS, Inc. and Contributors
+
+#if WITH_EDITOR
+
+#include "CesiumGlobeAnchorComponent.h"
+#include "CesiumTestHelpers.h"
+#include "Editor.h"
+#include "Engine/World.h"
+#include "EngineUtils.h"
+#include "GlobeAwareDefaultPawn.h"
+#include "Misc/AutomationTest.h"
+#include "Tests/AutomationEditorCommon.h"
+
+BEGIN_DEFINE_SPEC(
+    FGlobeAwareDefaultPawnSpec,
+    "Cesium.GlobeAwareDefaultPawn",
+    EAutomationTestFlags::ApplicationContextMask |
+        EAutomationTestFlags::ProductFilter)
+
+FDelegateHandle subscriptionPostPIEStarted;
+
+END_DEFINE_SPEC(FGlobeAwareDefaultPawnSpec)
+
+void FGlobeAwareDefaultPawnSpec::Define() {
+  Describe(
+      "should not spike altitude when very close to final destination",
+      [this]() {
+        LatentBeforeEach(
+            EAsyncExecution::TaskGraphMainThread,
+            [this](const FDoneDelegate& done) {
+              UWorld* pWorld = FAutomationEditorCommonUtils::CreateNewMap();
+
+              TSoftObjectPtr<UObject> DynamicPawn =
+                  TSoftObjectPtr<UObject>(FSoftObjectPath(TEXT(
+                      "Class'/CesiumForUnreal/DynamicPawn.DynamicPawn_C'")));
+              AGlobeAwareDefaultPawn* pPawn =
+                  pWorld->SpawnActor<AGlobeAwareDefaultPawn>(
+                      Cast<UClass>(DynamicPawn.LoadSynchronous()));
+
+              subscriptionPostPIEStarted =
+                  FEditorDelegates::PostPIEStarted.AddLambda(
+                      [done](bool isSimulating) { done.Execute(); });
+              FRequestPlaySessionParams params{};
```

Comment:
I noticed a bit of audio logging spam to the console and report log with this test. I tried running the PIE without sound, but noticed no change

```
              ULevelEditorPlaySettings* playSettings = NewObject<ULevelEditorPlaySettings>();
              playSettings->EnableGameSound = false;

              FRequestPlaySessionParams params{};
              params.EditorPlaySettings = playSettings;
```

I wonder if we can just disable audio logging for the test completely?

Reply:
I'm not sure. I think it's possible to mark certain messages as expected, but I don't think that would be appropriate here.

Reply:
I'm not sure either. Maybe just something to keep in mind in the future, if excess test logging starts causing problems.